Composer/packages/client/src/pages/botProject/runtime-settings/RuntimeSettings.tsx

```diff
@@ -191,8 +204,8 @@ export const RuntimeSettings: React.FC<RouteComponentProps<{ projectId: string }
           errorMessage={formDataErrors.path}
           label={formatMessage('Runtime code location')}
           styles={name}
-          value={settings.runtime ? settings.runtime.path : ''}
-          onChange={updateSetting('path')}
+          value={runtimePath}
+          onChange={updateSetting(RuntimeType.Path)}
           onRenderLabel={onRenderLabel}
         />
```

Comment:
we can probably add a onBlur here, set setting only once.

Reply:
> we can probably add a onBlur here, set setting only once.

Sync with other setting property.  Just update property when onBlur the text area.